Flexible Learning/units_manage_addProcess.php

```diff
@@ -75,20 +84,45 @@
         $file = $_FILES['file'] ?? null;
 
         // Upload the file, return the /uploads relative path
+<<<<<<< Updated upstream
         $data['logo'] = $fileUploader->uploadFromPost($file, $name);
+=======
+        $data['logo'] = $fileUploader->uploadFromPost($file, $name, $data['name']);
+>>>>>>> Stashed changes
 
         if (empty($data['logo'])) {
             $partialFail = true;
         }
     }
-    
+<<<<<<< Updated upstream
+
+=======
+
+>>>>>>> Stashed changes
     // Create the record
     if (!$flexibleLearningUnitID = $unitGateway->insert($data)) {
         $URL .= '&return=error2';
         header("Location: {$URL}");
         exit;
     }
 
+<<<<<<< Updated upstream
```

Comment:
You've got quite a few merge issues, relating to upstream and stash. I think @SKuipers will be able to advise you on dealing with these.